admin/create_credentials.R

```diff
@@ -0,0 +1,9 @@
+# Code to create the credential files
```

Comment:
in the projects I use this. I don't commit this file as you could end commiting the login details. I would add it to the gitignore and then share the file with whoever needs to deploy the app. As you have already commited, two options, delete this branch and recreate somewhere else, or delete this file, commit, then add to gitignore and add back to the project. Hope that makes sense

Reply:
Thanks @jvillacampa, I'll try deleting the file, commit, and add it to the gitignore